app/models/partner.rb

```diff
@@ -17,6 +17,7 @@ class Partner < ApplicationRecord
   # Associations
   has_and_belongs_to_many :users
   has_many :calendars, dependent: :destroy
+  has_many :calendar_places, class_name: 'Calendar', foreign_key: :place_id, inverse_of: :place
```

Comment:
This place / partner muddling is getting pretty confusing (for me at least). I'm wondering if there's some way we can make it clearer? 

I'm not entirely sure what this represents. the name suggests it's other partners(places)? that have been linked to the calendar the partner owns but the test set up makes it look like it's a list of calendars.

Is it all the calendars that list this partner as the location of their events, rather than the partner the calendar belongs to?

Reply:
I agree on this but think we can handle them separately - I've made this ticket #2268 to document